src/CTDirect.jl

```diff
@@ -39,5 +39,7 @@ export export_OCP_solution
 export read_OCP_solution
 export OCP_Solution_discrete
 
+export _OptimalControlInit #temp
```

Comment:
Yup, still needed in the rather uncommon case where a user wants to call *solve* on an existing **DOCP** with an explicit init that will override the init already in the DOCP. We could remove this possibility and have the user first change the initial in the DOCP with *setInitialGuess(docp, init)*, then solve it.

But what I meant by temp is that the export will eventually be in *CTBase* once the initialisation part is updated there. I can prefix the 3 calls in *CTDirect* if we don't like exporting an 'internal' constructor.

Reply:
To sum up: *_OptimalControlInit* is internal since most functions taking an initial guess now receive the *arguments* to build an **_OptimalControlInit** instead of an **_OptimalControlInit** itself, the call to the constructor being made internally only when absolutely needed. This is what allows the nice syntax *init=whatever*. To sum up, *_OptimalControlInit*
- is defined and exported in *CTBase*
- is called 3 times inside *CTDirect*
- is not called in *OptimalControl* (we now just use __ocp_init() = () without the constructor)

Reply:
@PierreMartinon OK. For now, as it is not called nor, *a fortiori*, exported by `OptimalControl.jl` not a big deal.